Notify/Notify/Notify.Android/MainActivity.cs

```diff
@@ -38,15 +34,29 @@ protected override void OnCreate(Bundle savedInstanceState)
             {
                 Intent intent = new Intent(Android.Provider.Settings.ActionManageOverlayPermission);
                 intent.SetFlags(ActivityFlags.NewTask);
-                this.StartActivity(intent);
+                StartActivity(intent);
             }
 
-            LoadApplication(new App());
-        }
+            var app = new App();
+            
+            if (Preferences.ContainsKey("NotifyUserName") && Preferences.ContainsKey("NotifyPassword"))
+            {
+                Debug.WriteLine($"logged in with credentials from preferences.\n UserName: {Preferences.Get("NotifyUserName", string.Empty)}" +
```

Comment:
1. An extra space after "\n".
2. Use Environment.NewLine instead of "\n".
3. Uppercase in "logged".
4. I think it should be "Logging in with credentials..", and add a TODO after this log for implementing the actual login implementation.